test/JDBC/expected/BABEL-3147-vu-prepare.out

```diff
@@ -1,8 +1,13 @@
+-- error should not have been thrown, should be fixed under BABEL-1710
```

Comment:
So this started breaking now for CHAR datatype because we are using `babelfish_conv_helper_to_varchar` function similar to VARCHAR datatype?

Reply:
Yes